lib/components/MarketplaceAppVisibility/index.ts

```diff
@@ -1 +1,9 @@
-export { default } from './MarketplaceAppVisibility'
+import { connect } from 'react-redux'
+
+import MarketplaceAppVisibility from './MarketplaceAppVisibility'
+
+export const mapStateToProps = ({ profile }) => ({
+  userRole: profile.profile.current_org.role.name,
```

Comment:
needs to be ready for users without organization (baseUser)

Reply:
Users need to have (or be associated to) an organisation in order to be able to create/edit apps, and thus, their labels, so the described scenario is fortunately covered for.